src/containers/network/NetworkMembers.js

```diff
@@ -1,3 +1,67 @@
 import React from 'react'
-const NetworkMembers = props => <span />
-export default NetworkMembers
+import { prefetch } from 'react-fetcher'
+import { connect } from 'react-redux'
+import { debounce } from 'lodash'
+import { fetchPeople } from '../../actions/fetchPeople'
+import { fetchWithCache, connectedListProps, refetch } from '../../util/caching'
+import ScrollListener from '../../components/ScrollListener'
+import PersonCards from '../../components/PersonCards'
+const { array, bool, func, number, object } = React.PropTypes
+
+const subject = 'network'
+const fetch = fetchWithCache(fetchPeople)
+
+@prefetch(({ dispatch, params: { id }, query }) => dispatch(fetch(subject, id, query)))
+@connect((state, { params: { id }, location: { query } }) => {
+  return {
+    ...connectedListProps(state, {subject, id, query}, 'people'),
+    network: state.networks[id],
+    currentUser: state.people.current
+  }
+})
+export default class NetworkMembers extends React.Component {
```

Comment:
this can be stateless too; you can define `loadMore` and `updateQuery` inside `render`